packages/dam-app-base/src/AppConfig/FieldSelector.tsx

```diff
@@ -33,28 +32,29 @@ export default class FieldSelector extends React.Component<Props> {
     const { compatibleFields, contentTypes, selectedFields } = this.props;
 
     return (
-      <Typography>
+      <React.Fragment>
```

Comment:
Looks like this fragment is not needed

Reply:
It is. An array is not a valid React node

Reply:
oh, you're right.